### PR TITLE
Use library path and clear most of the UDL file

### PR DESCRIPTION
**Pull request by aringenbach**
This is an easy first step towards getting rid of the UDL file and use uniffi attributes to generate the bindings code.
Mostly checking that CI is happy with that change.

### CODE REVIEW

File: platforms/android/library/build.gradle [hunk 1]
```diff
@@ -95,7 +95,7 @@ dependencies {
 android.libraryVariants.all { variant ->
     def generateBindingTasks = tasks.register("generate${variant.name.capitalize()}UniFFIBindings", Exec) {
         workingDir "${project.projectDir}"
-        commandLine 'cargo', 'uniffi-bindgen', 'generate', '../../../bindings/wysiwyg-ffi/src/wysiwyg_composer.udl', '--language', 'kotlin', '--out-dir', "${buildDir}/generated/source/uniffi/${variant.name}/java"
+        commandLine 'cargo', 'uniffi-bindgen', 'generate', '--library', "${buildDir}/rustJniLibs/android/x86_64/libuniffi_wysiwyg_composer.so", '--language', 'kotlin', '--out-dir', "${buildDir}/generated/source/uniffi/${variant.name}/java"
```
[the remark this answers is not in the document]
Comment (jonnyandrew) [reply platforms/android/library/build.gradle:98]: We are currently building for arm, arm64, x86 and x86_64 targets (configured [here](https://github.com/matrix-org/matrix-rich-text-editor/pull/749/files#diff-9a556f815aed9de878120ac784c38be9896b55d362592e0281ae3acdb757b66bR19)).

Are you able to access those through `cargo.targets` in this context? If not, perhaps we can extract these to a top level variable.

Also, note that the directory names don't seem to match the target names for the arm targets.

[the remark this answers is not in the document]
Comment (jonnyandrew) [reply platforms/android/library/build.gradle:98]: (I guess the command should fail if this directory doesn't exist anyway? In which case I think we can also leave this hard-coded, perhaps with a comment that the target architecture of the arg supplied to `--library` is arbitrary)

[the remark this answers is not in the document]
Comment (jonnyandrew) [reply platforms/android/library/build.gradle:99]: I had a look at this and https://github.com/matrix-org/matrix-rich-text-editor/pull/753 seems to do the trick

[the remark this answers is not in the document]
Comment (aringenbach) [reply platforms/android/library/build.gradle]: Cool, I just merged it. I let the CI run here, and then this change should be ready to fly

File: bindings/wysiwyg-ffi/src/wysiwyg_composer.udl [hunk 2]
```diff
@@ -12,155 +12,4 @@
...
-};
-
-[Error]
-enum DomCreationError {
-    "MarkdownParseError",
-    "HtmlParseError",
-};
-
-
+namespace wysiwyg_composer {};
```
Comment (jonnyandrew) [inline bindings/wysiwyg-ffi/src/wysiwyg_composer.udl:15]: :tada:

**Review by jonnyandrew — APPROVED** [review]
Nice work!

**Pull request merged**